### DISCUSSION TITLE
Does a running xpra session have the ability to print out the the running options?

### DISCUSSION THREAD

**Discussion by vonHabsi** [Xpra-org/xpra#3821] [General]
Does a running xpra session have the ability to print out the running options?

This must include those used on the command invocation, those chosen automatically to suit runtime conditions, those dependent on features offered by the operating system's physical and software environment and those unchanged from the defaults.

And I assume they will differ on the client and the server.

**Comment by totaam** [comment]
> Does a running xpra session have the ability to print out the running options?

You mean for an existing session? one started with `xpra start` for example?
Then yes:
```shell
xpra info | grep server.argv=
```
See also:
```shell
$ cat /var/run/user/1000/xpra/$ID/config
$ cat /var/run/user/1000/xpra/$ID/cmdline
```
Where `$ID` is the display if your session.

**Reply by vonHabsi** [reply]
How can I check the codec used to transmit sound or the compression level?

In my current session sound quality is poor although both client and server are on the same gigabit LAN.

Is there some way to set the codec and compression level used for sound?

**Reply by totaam** [reply]
> How can I check the codec used to transmit sound or the compression level?

Try:
```shell
$ xpra info | grep client.sound.speaker
```

We don't really set the compression level.
You haven't specified your distro or what packages are installed, so it's impossible to guess which codecs are available.

> Is there some way to set the codec and compression level used for sound?

Try: `--speaker-codec=help`
